ebos/eclgenericwriter.cc

```diff
@@ -463,6 +473,27 @@ doWriteOutput(const int                     reportStepNum,
         restartValue.addExtra("OPMEXTRA", std::vector<double>(1, nextStepSize));
     }
 
+    // Add nnc flows and flores is existing.
+    if (isFlowsn) {
+        const auto flowsn_global = isParallel ? this->collectToIORank_.globalFlowsn() : std::move(flowsn);
+        for (const auto& flows : flowsn_global) {
+            if (flows.first.empty())
+                continue;
+            restartValue.addExtra(flows.first, UnitSystem::measure::rate, flows.second.second);
```

Comment:
I think the unit here is wrong, the unit here is for reservoir volumes. The logic here and on lines 490--494 should be swapped. Testing with field units should show the problem.